content/contributor/runtime/execution.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Execution"
+sidebartitle: "Execution"
+url: "/contributor/runtime/execution"
+weight: 3
+---
+
+The runtime treats [scenes]({{< ref "/contributor/content/entity-types/scenes" >}}) like typical Node modules. Once the sandbox is set up (i.e. all [globals]({{< relref "" >}}) are injected and [modules]({{< relref "modules" >}}) are ready to be loaded), the code of the scene will be evaluated so it can populate the [exports]({{< relref "globals#module" >}}) object.
+
+Two methods will be picked up by the host from `exports`: [`onStart`](#onStart) (optional) and [`onUpdate`](#onUpdate) (mandatory).
+
+```ts
+type Exports = {
+  onStart?: () => Promise<void>
+  onUpdate: (deltaSeconds: number) => Promise<void>
+}
+```
+
+{{< info >}}
+Most scenes use the [Decentraland SDK]({{< ref "/creator/sdk7/getting-started/sdk-101" >}}), which provides a higher-level interface and automatically exports these methods. However, the SDK is _not_ part of the protocol, and scene developers can decide not to use it.
+{{< /info >}}
+
+!!add nice diagram, light on detail
+
+During the life-cycle of a scene, the runtime will ensure that calls to these methods are never made concurrently. The returned `Promise` will always be awaited before the scene receives a new call.
+
+
+###### `onStart` {#onStart}
+
+The life-cycle of a scene begins with the asynchronous `onStart` method. It's the place to make one-time initializations that must complete before the first call to [`onUpdate`](#onUpdate) is made.
+
+```ts
+onStart(): Promise<void>
+```
+
+Exporting `onStart` is not required, and scenes are not expected to do anything in particular if they do. The method can be missing or do nothing and return an immediately resolved `Promise`.
+
+{{< info >}}
+Scenes using the Decentraland SDK export an empty `onStart` method, unless explicitly overridden by the developer.
```

Comment:
This is no-longer the case. The SDK adds its own logic to onStart

Reply:
This wasn't ready for review, I pushed accidentally